efi-log-check.sh

```diff
@@ -0,0 +1,33 @@
+SYSROOT=/mnt/sysroot
+RESULT_FILE=${SYSROOT}/root/RESULT
+
+# Check if log util can be called
+function check_log_util_exist() {
+    if [ ! -f /usr/libexec/anaconda/log-capture ]; then
+        echo "*** Failed check: log-capture util does not exist" >> ${RESULT_FILE}
+    fi
+}
+
+# Check if log utils runs without failure
+function check_log_util_runs() {
+    local exit_code=0
+
+    # Trigger the log util
+    /usr/libexec/anaconda/log-capture
+
+    if [[ $? -ne ${exit_code} ]]; then
+        echo "*** Failed check: log-capture util cant be called" >> ${RESULT_FILE}
```

Comment:
I'd maybe rather say it failed and put the exit code value in. But I don't insist.

Reply:
Added